entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+TRACEE_EBPF_EXE=${TRACEE_EBPF_EXE:="/tracee/tracee-ebpf"}
+TRACEE_MOUNT_EXE=${TRACEE_MOUNT_EXE:="mount"}
+TRACEE_WEBHOOK_CONFIG=${TRACEE_WEBHOOK_CONFIG:="/tmp/tracee/integrations-config.yaml"}
+TRACEE_WEBHOOK_EXE=${TRACEE_WEBHOOK_EXE:="/tracee/falcosidekick"}
+TRACEE_RULES_EXE=${TRACEE_RULES_EXE:="/tracee/tracee-rules"}
+
+$TRACEE_MOUNT_EXE | grep -q 'debugfs on /sys/kernel/debug'
+if ! $TRACEE_MOUNT_EXE | grep -q 'debugfs on /sys/kernel/debug'; then
+  $TRACEE_MOUNT_EXE -t debugfs debugfs /sys/kernel/debug/
+fi
+
+if [ "$1" = "trace" ]; then
+	shift
+	$TRACEE_EBPF_EXE $@
+	exit
+fi
+
+if [ "$1" = "--rules" ]; then
+	flag_rules="$1=$2"
+fi
+
+if [ -f $TRACEE_WEBHOOK_CONFIG ]; then
+	flag_webhook_config="--config-file=$TRACEE_WEBHOOK_CONFIG"
+fi
+
+if $TRACEE_WEBHOOK_EXE $flag_webhook_config& then
+	flag_webhook_url="--webhook=http://localhost:2801"
+fi
+
+$TRACEE_EBPF_EXE --output=format:gob --output=option:eot | $TRACEE_RULES_EXE --tracee-file=/dev/stdin $flag_webhook_url $flag_rules
```

Comment:
did you read the PR message? I'm waiting for it to be merged then I'll make this ready.

Is there a problem with stdin? why would it not scale?

Reply:
`why would it not scale?`
As we already discussed, adding other sources will require the usage of pipes, and can't use stdin in parallel

`Is it something that would stop the MVP? Or can be addressed later?`
Doesn't have to stop MVP, but it feels like a regression to me.
We had file input (not stdin), which could have supported streaming input source before the removal of eot.
Now we took step back - and only stdin supports streaming

Reply:
> adding other sources will require the usage of pipes, and can't use stdin in parallel

oh ok.. but we have just one input source now, and file is not a viable option (see below)

> We had file input (not stdin), which could have supported streaming input source before the removal of eot.
Now we took step back - and only stdin supports streaming

I don't think it's a step back because the file based solution was practically unusable (file would indefinitely grow with no way to trim or partition it).

Completely agree that the best way forward is something like named pipes, but we want to release soon and for the sake of the MVP, I think this can be good enough.

Reply:
Agree. For MVP that's enough.

> I don't think it's a step back because the file based solution was practically unusable (file would indefinitely grow with no way to trim or partition it).

File doesn't have to grow indefinetly. it's maximum size can be specified and drop events when full (like a circular buffer). But I don't think that going forward this is the correct solution anyways, I just wanted to make the point that using stdin would not scale